packages/design-tokens/src/tokens/functional/components/logosuite/colors.json

```diff
@@ -11,6 +11,16 @@
             "value": "var(--brand-color-text-default)",
             "dark": "var(--brand-color-text-default)"
           }
+        },
+        "image": {
```

Comment:
Thinking this should either be called `--brand-LogoSuite-color-logo-filter` or be placed straight into the CSS file 🤔. It currently overlaps with the existing token name, which is still valid.

If put straight to code, we could override the original token for the `img` selector only and apply these styles directly into the stylesheet.

Reply:
Could you expand on this a little please?

Are you suggesting we update `colors.json` to be as follows:

```json
{
  "brand": {
    "LogoSuite": {
      "color": {
        "logo": {
          "muted": {
            "value": "var(--brand-color-text-muted)",
            "dark": "var(--brand-color-text-muted)"
          },
          "emphasis": {
            "value": "var(--brand-color-text-default)",
            "dark": "var(--brand-color-text-default)"
          },
          "filter": {
            "muted": {
              "value": "brightness(0) saturate(100%) invert(37%) sepia(7%) saturate(778%) hue-rotate(171deg) brightness(96%) contrast(91%)",
              "dark": "brightness(0) saturate(100%) invert(61%) sepia(8%) saturate(430%) hue-rotate(171deg) brightness(94%) contrast(92%)"
            },
            "emphasis": {
              "value": "brightness(0) saturate(100%) invert(12%) sepia(20%) saturate(450%) hue-rotate(172deg) brightness(100%) contrast(91%)",
              "dark": "brightness(0) saturate(100%) invert(91%) sepia(2%) saturate(2455%) hue-rotate(193deg) brightness(107%) contrast(98%)"
            }
          }
        }
      }
    }
  }
}
```

For the alternative you suggested, could you point me in the direction of the token it overlaps with please? I'm struggling to find it.

Thanks! 🙂

Reply:
> Are you suggesting we update colors.json to be as follows

Yep, exactly that, because I see filter as an implementation child of logo, not an alternative / sibling.